lib/common/grpc-service.js

```diff
@@ -241,6 +234,21 @@ GrpcService.prototype.request = function(protoOpts, reqOpts, callback) {
     grpcOpts.deadline = new Date(Date.now() + protoOpts.timeout);
   }
 
+  var isStreamRequest = false;
+
+  if (protoOpts.stream) {
+    isStreamRequest = true;
+    delete grpcOpts.stream;
+  }
+
+  if (isStreamRequest) {
```

Comment:
BigTable has introduced a gRPC streaming endpoint, which I believe is a first for us. This still needs some work (I don't think just doing a concat stream here would be very efficient/wise/etc.), however my initial thought is just to pass in a `stream` option so we can differentiate between the two.